src/monattack.cpp

```diff
@@ -2119,15 +2119,6 @@ bool mattack::impale( monster *z )
                                        z->name() );
 
         target->on_hit( z, bodypart_id( "torso" ),  z->type->melee_skill );
-        if( one_in( 60 / ( dam + 20 ) ) ) {
```

Comment:
why remove this if the other monsters keep their bleed?

Reply:
Long swipe does little damage while it's directed at the neck (arteries), so I'm under impression that extra bleeding was the desired effect. Bio_op impale has extra bleed only on crit. So since impale does a quite significant DT_STAB damage that will trigger bleeding by itself (while  it wouldn't before), I thought that stacking additional extra bleeding here would be an overkill. What do you think?